src/DataViews/include/DataViews/LiveFunctionsDataView.h

```diff
@@ -84,6 +82,8 @@ class LiveFunctionsDataView : public DataView {
   };
 
  private:
+  [[nodiscard]] const orbit_client_protos::FunctionInfo* GetFunctionInfoFromRow(int row) override;
```

Comment:
Couldn't this function be marked `const`?

Reply:
Thanks for pointing out this. `SamplingReportDataView::GetFunctionInfoFromRow` updates `SamplingDataView::functions_`, so this cannot be marked `const`.

Reply:
Ah I see. Thanks for the explanation. LGTM!